common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerResult.java

```diff
@@ -85,6 +84,8 @@
         static final String CLI_TELEM_SUB_ERROR_CODE = "cli_telem_suberror_code";
     }
 
+    private static final long serialVersionUID = 8606631820514878489L;
```

Comment:
What is this for?

Reply:
Fixed serialVersionUID is needed for back-compat. Otherwise, the serialVersionUID will change every time when any code changing in the serializable class.